ui/components/multichain/create-solana-account-modal/create-solana-account-modal.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
```

Comment:
Doesnt this already exist as `create-snap-account`?

Reply:
Actually this one is using `create-snap-account`, it's just a modal that uses it, it was needed as we don't have modal open, as we had in case of network or account modal flow.